server/room-battle.js

```diff
@@ -47,13 +47,16 @@ class RoomBattlePlayer {
 		user.updateSearch();
 
 		this.slot = slot;
-		this.slotNum = Number(slot.charAt(1)) - 1;
+		const channelIndex = Number(slot.charAt(1));
+		this.slotNum = channelIndex - 1;
+		this.channelIndex = /** @type {0 | 1 | 2 | 3 | 4} */
+			(game.gameType === 'multi' ? channelIndex % 2 + 1 : channelIndex);
```

Comment:
I think I still need the + 1 here (even though I've removed it from `channelIndex` because its not longer based on `slotNum`) because anything % 2 is going to be 0  | 1?

Reply:
`%2+1` will reverse the channels. I'd honestly rather figure this out from the `slotNum`, even if it's a bit silly to subtract 1 and then add 1.

Reply:
Done.